drivers/wifi/nrf700x/zephyr/src/zephyr_work.c

```diff
@@ -21,9 +21,12 @@
 
 LOG_MODULE_DECLARE(wifi_nrf, CONFIG_WIFI_LOG_LEVEL);
 
-K_THREAD_STACK_DEFINE(wq_stack_area, CONFIG_NRF700X_WORKQ_STACK_SIZE);
+K_THREAD_STACK_DEFINE(wq_stack_area, 1024);
```

Comment:
Rename wq_stack_area to bh_wq_stack_area since we are now differentiating workqueues as IRQ and BH workqueues.